circle.yml

```diff
@@ -50,13 +50,13 @@ dependencies:
     - docker-compose version
     - docker version
   override:
-    - .circle/docker-compose.sh pull
+    - .circle/docker-compose.sh pull ${DISTRO}
   post:
-    - .circle/docker-compose.sh build
+    - .circle/docker-compose.sh build ${DISTRO}
```

Comment:
Yeah, I need this.
It reverts old things before direct push. There is also file renaming "refactoring" coming from that, but I'll synchronize with your other changes in `st2`.